Serializer/SymfonySerializerAdapter.php

```diff
@@ -59,7 +59,9 @@ private function convertContext(Context $context)
             $newContext[$key] = $value;
         }
 
-        $newContext['groups'] = $context->getGroups();
+        if (count($context->getGroups())) {
```

Comment:
I wonder if we should allow to set the groups in the context to `null` to make it possible to distinguish the case where someone builds their group dynamically and actually wants to force the serializer to exclude everything in that case.